pkg/api/latest/dynakube/dynakube_status.go

```diff
@@ -42,7 +42,9 @@ type DynaKubeStatus struct { //nolint:revive
 	UpdatedTimestamp metav1.Time `json:"updatedTimestamp,omitempty"`
 
 	// Observed state of Dynatrace API
-	DynatraceApi DynatraceApiStatus `json:"dynatraceApi,omitempty"`
+	// TODO: Rename printer column to match Go variable name (currently kept as-is to ensure backwards compatibility)
+	// +kubebuilder:printcolumn:name="dynatraceApi",type=string,JSONPath=`.spec.status.dynatraceApi`
```

Comment:
```suggestion
```

The TODO contradicts the fact that you introduced this `// +kubebuilder:...` here

There are maaany other places where these would need to be added
- and there is already a ticket for it [DAQ-1350](https://dt-rnd.atlassian.net/browse/DAQ-1350)
   - it is old and it is mainly OLM related
   - may have affect on the desc of the CRD, however, we are cutting smaller and smaller by the day, so I see little value in them

also the "correct" thing would be:
```
// +kubebuilder:printcolumn:name="dynatraceApi",type=string,JSONPath=`.status.dynatraceApi`
```

Reply:
Thanks, applied in 31efe62d